onnxruntime/core/providers/webnn/builders/helper.h

```diff
@@ -280,8 +280,7 @@ static const InlinedHashMap<std::string, std::string> op_map = {
     {"Trilu", "triangular"},
     {"Unsqueeze", "reshape"},
     {"Where", "where"},
-    {"Xor", "logicalXor"}
-};
+    {"Xor", "logicalXor"}};
```

Comment:
Hmm, that's inconsistently odd, as we don't put the closing brace on the same line in most *other* places. e.g. This doesn't happen:

```
inline bool IsEmptyTensor(const InitializedTensorSet& initializers, const std::string& name) {
  return std::any_of(dims.begin(), dims.end(), [](auto d) { return d == 0; }); }
```

To reduce diffing noise in the future when inserting/deleting/reordering, what if we just put a comma after the last item?

```suggestion
    {"Xor", "logicalXor"},
};
```

Reply:
Done.